perllib/FixMyStreet/Script/Alerts.pm

```diff
@@ -132,6 +133,18 @@ sub send() {
                 }
             }
 
+            # This seems a bit inefficient to do for every single update, just
+            # so we can later determine if it came from a staff/superuser. It
+            # is run as a batch job though so maybe that doesn't matter?
+            # Would a JOIN in the query be better option, or maybe adding something
+            # to the comment's extra when it's created? The latter approach would
+            # also solve the theoretical problem of a user leaving an HTML update
```

Comment:
All updates left by staff should have extra data of is_body_user or is_superuser so that it doesn't matter if the user loses their staff status later. Also means you don't need to get the user here, so that's handy!

Reply:
That is handy indeed, I'll use that 👍